resources/js/components/pages/auth/Login.vue

```diff
@@ -0,0 +1,68 @@
+                    </a>
+                </p>
+            </div>
+            <form class="mt-8 space-y-6 border-2 rounded-2xl bg-white p-6" method="POST" action="/login">
+                <input type="hidden" name="_token" :value="token">
+                <div class="rounded-md shadow-sm -space-y-px">
+                    <div class="mb-2">
+                        <label for="email-address" class="mb-1 text-sm font-sans font-bold">Email address</label>
+                        <input id="email-address" name="email" type="email" autocomplete="email" required="" class=" appearance-none rounded relative block w-full px-3 py-2 border border-gray-300 placeholder-gray-500 text-gray-900  focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 focus:z-10 sm:text-sm" placeholder="Email address" />
+                    </div>
+                    <div class="mt-3">
+                        <label for="password" class="mb-1 text-sm font-sans font-bold">Password</label>
+                        <input id="password" name="password" type="password" autocomplete="current-password" required="" class=" appearance-none rounded relative block w-full px-3 py-2 border border-gray-300 placeholder-gray-500 text-gray-900 focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 focus:z-10 sm:text-sm" placeholder="Password" />
+                    </div>
+                </div>
+
+                <div class="flex items-center justify-between">
+                    <div class="flex items-center">
+                        <input id="remember-me" name="remember-me" type="checkbox" class="h-4 w-4 text-indigo-600 focus:ring-indigo-500 border-gray-300 rounded" />
+                        <label for="remember-me" class="ml-2 block text-sm text-gray-900">
+                            Remember me
+                        </label>
+                    </div>
+
+                    <div class="text-sm">
+                        <a href="#" class="font-medium text-indigo-600 hover:text-indigo-500">
+                            Forgot your password?
+                        </a>
+                    </div>
+                </div>
+
+                <div>
+                    <button type="submit" class="group relative w-full flex justify-center py-2 px-4 border border-transparent text-sm font-medium rounded-md text-white bg-indigo-600 hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500">
+            <span class="absolute left-0 inset-y-0 flex items-center pl-3">
+              <LockClosedIcon class="h-5 w-5 text-indigo-500 group-hover:text-indigo-400" aria-hidden="true" />
+                    </span>
+                        Sign in
+                    </button>
+                </div>
+            </form>
+        </div>
+    </div>
+</template>
+
+<script>
+import { LockClosedIcon } from '@heroicons/vue/solid'
+
+export default {
+    components: {
+        LockClosedIcon
+    },
+    props: ['postUrl', 'token'],
```

Comment:
```suggestion

    props: ['postUrl', 'token']
```

yukarı boşluk sonraki , kalkacak